concepts/Common/Licensing/40 Troubleshooting.md

```diff
@@ -0,0 +1,5 @@
+If you have problems verifying your license key, try downloading it again in [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/) and specifying it in your application config.
```

Comment:
Download manager will only have the option to copy the license key to the clipboard. 
downloading it -> copy / copying it